nautobot/dcim/forms.py

```diff
@@ -567,7 +567,6 @@ def __init__(self, data=None, *args, **kwargs):
         super().__init__(data, *args, **kwargs)
 
         if data:
-
```

Comment:
Did `black` require these whitespace changes? If they're not required to pass linting, I'd prefer to see these reverted since they just add noise to the diffs.

Reply:
Nope, Illd revert the changes